pkg/autoscaler/collector.go

```diff
@@ -312,34 +330,27 @@ func (c *collection) currentMetric() *av1alpha1.Metric {
 func (c *collection) record(stat Stat) {
 	// Proxied requests have been counted at the activator. Subtract
 	// them to avoid double counting.
-	c.concurrencyBuckets.Record(stat.Time, stat.AverageConcurrentRequests-stat.AverageProxiedConcurrentRequests)
+	c.concurrencyBuckets.Record(stat.Time,
+		stat.AverageConcurrentRequests-stat.AverageProxiedConcurrentRequests)
```

Comment:
nit: Only do this computation once?

i.e.

```golang
concurrency := stat.AverageConcurrentRequests-stat.AverageProxiedConcurrentRequests
rps := stat.RequestCount-stat.ProxiedRequestCount
```

Reply:
I think compiler should optimize for this, but I can do it for readability :)